pkg/cmd/roachtest/gorm_helpers.go

```diff
@@ -0,0 +1,138 @@
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package main
+
+const gormTestHelperGoFile = `package tests_test
+
+import (
+	"fmt"
+	"log"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"time"
+
+	"gorm.io/driver/mysql"
+	"gorm.io/driver/postgres"
+	"gorm.io/driver/sqlite"
+	"gorm.io/driver/sqlserver"
+	"gorm.io/gorm"
+	"gorm.io/gorm/logger"
+	. "gorm.io/gorm/utils/tests"
+)
+
+var DB *gorm.DB
+
+func init() {
+	var err error
+	if DB, err = OpenTestConnection(); err != nil {
+		log.Printf("failed to connect database, got error %v", err)
+		os.Exit(1)
+	} else {
+		sqlDB, err := DB.DB()
+		if err == nil {
+			err = sqlDB.Ping()
+		}
+
+		if err != nil {
+			log.Printf("failed to connect database, got error %v", err)
+		}
+
+		RunMigrations()
+		if DB.Dialector.Name() == "sqlite" {
+			DB.Exec("PRAGMA foreign_keys = ON")
+		}
+	}
+}
+
+func OpenTestConnection() (db *gorm.DB, err error) {
+	dbDSN := os.Getenv("GORM_DSN")
+	switch os.Getenv("GORM_DIALECT") {
+	case "mysql":
+		log.Println("testing mysql...")
+		if dbDSN == "" {
+			dbDSN = "gorm:gorm@tcp(localhost:9910)/gorm?charset=utf8&parseTime=True&loc=Local"
+		}
+		db, err = gorm.Open(mysql.Open(dbDSN), &gorm.Config{})
+	case "postgres":
+		log.Println("testing postgres...")
+    port, ok := os.LookupEnv("PGPORT")
+    if !ok {
+      port = "9920"
+    }
+    user, ok := os.LookupEnv("PGUSER")
+    if !ok {
+      user = "root"
+    }
+    password, ok := os.LookupEnv("PGPASSWORD")
+    if !ok {
+      password = ""
+    }
+		if dbDSN == "" {
+			dbDSN = fmt.Sprintf("user=%s password=%s dbname=gorm host=localhost port=%s sslmode=disable TimeZone=Asia/Shanghai", user, password, port)
+		}
+		db, err = gorm.Open(postgres.New(postgres.Config{
+			DSN:                  dbDSN,
+			PreferSimpleProtocol: true,
+		}), &gorm.Config{})
+	case "sqlserver":
+		// CREATE LOGIN gorm WITH PASSWORD = 'LoremIpsum86';
+		// CREATE DATABASE gorm;
+		// USE gorm;
+		// CREATE USER gorm FROM LOGIN gorm;
+		// sp_changedbowner 'gorm';
+		// npm install -g sql-cli
+		// mssql -u gorm -p LoremIpsum86 -d gorm -o 9930
+		log.Println("testing sqlserver...")
+		if dbDSN == "" {
+			dbDSN = "sqlserver://gorm:LoremIpsum86@localhost:9930?database=gorm"
+		}
+		db, err = gorm.Open(sqlserver.Open(dbDSN), &gorm.Config{})
+	default:
+		log.Println("testing sqlite3...")
+		db, err = gorm.Open(sqlite.Open(filepath.Join(os.TempDir(), "gorm.db")), &gorm.Config{})
+	}
+
+	if debug := os.Getenv("DEBUG"); debug == "true" {
+		db.Logger = db.Logger.LogMode(logger.Info)
+	} else if debug == "false" {
+		db.Logger = db.Logger.LogMode(logger.Silent)
+	}
+
+	return
+}
+
+func RunMigrations() {
+	var err error
+	allModels := []interface{}{&User{}, &Account{}, &Pet{}, &Company{}, &Toy{}, &Language{}}
+	rand.Seed(time.Now().UnixNano())
```

Comment:
This is giving the linter trouble (wants timeutil.Now()), is there a way we can tell the linter to ignore this?